pkg/pillar/kubeapi/kubevirt.go

```diff
@@ -120,3 +120,71 @@ func (v *vmis) Delete(ctx context.Context, name string, options *metav1.DeleteOp
 		Do(ctx).
 		Error()
 }
+
+func KubeVirt(restClient *rest.RESTClient, namespace string) KubeVirtInterface {
+	return &kv{
+		restClient: restClient,
+		namespace:  namespace,
+		resource:   "kubevirts",
+	}
+}
+
+type kv struct {
+	restClient *rest.RESTClient
+	namespace  string
+	resource   string
+}
+
+// Get the KubeVirt from the cluster by its name and namespace
+func (o *kv) Get(name string, options *metav1.GetOptions) (*v1.KubeVirt, error) {
```

Comment:
What does `o` stand for?

Reply:
@naiming-zededa waiting for your comment here.

Reply:
this one I think we just copied from the vendor file into our side, 'o' is what they use:
https://github.com/kubevirt/client-go/blob/main/kubecli/kv.go#L57

Reply:
I think there should be a comment in the code that it has been copied from there - the license may ask for more, but I am not a lawyer ...

Reply:
@naiming-zededa , although this is a very simple function, we must be very careful while copying from other places. I know both Get() and Update() didn't do much, basically call SetGroupVersionKind(...), so there isn't too much to do differently. The kubecli is licensed under Apache 2.0 as well, I don't think such small functions (which you also changed) can be considered derivative work, but I agree with @christoph-zededa , would be good to have a comment like `Adapted from kubevirt (http://github.com/kubevirt).....`, something like that...

Reply:
Comment would be good.

But does it mean we also need to track whether there are any future bug fixes in that file? Or is this such a small subset of that kv.go that we expect it to not be subject to any future fixes?

Reply:
added comments